src/js/components/modal.js

```diff
@@ -0,0 +1,49 @@
+export class cookies {
+    $bodyEle = document.querySelector('body');
+    bodyOverlayClassName = "overlay";
+    hideClassName = 'hide';
+    showModalClassName = 'show';
+    localStorageKey = 'modal';
+    
+
+    constructor(modalName) {
+        this.modal = document.querySelector(`#${modalName}`);
+        this.closeButton = this.modal.querySelector('button');
+        this.bodyEle = document.querySelector('body');
+    }
+
+    LocalSt() {
+        localStorage.setItem(this.localStorageKey, "accepted");
+    }
+
+    checkLocalStorage () {
+        localStorage.getItem(this.localStorageKey);
+        return;
```

Comment:
nic nie zwracasz, przenies tutaj kod z linii 20 i daj:

return localStorage.getItem(this.localStorageKey) === null;

dzieki temu ta funkcja bedzie od razu zwracac informacje true albo false